CodeComplianceTest_Engine/Query/Checks/MethodNameStartsUpper.cs

```diff
@@ -36,7 +36,7 @@ public static partial class Query
         [Message("Input attribute should start with a lowercase letter")]
```

Comment:
This message is wrong - out of scope change I know but could we fix it while we're here?